tests/integration/helpers.py

```diff
@@ -139,6 +140,50 @@ def restart_unit(model_full_name: str, unit: str) -> None:
     )
 
 
+@retry(
+    stop=stop_after_attempt(3),
+    wait=wait_fixed(5),
+    retry_error_callback=lambda _: False,
+    retry=lambda x: x is False,
+)
+def dashboard_unavailable(host: str, https: bool = False) -> bool:
+    try:
+        # Normal IP address
+        socket.inet_aton(host)
+    except OSError:
+        socket.inet_pton(socket.AF_INET6, host)
+        host = f"[{host}]"
+
+    protocol = "http" if not https else "https"
+    url = f"{protocol}://{host}:5601/auth/login"
+    arguments = {"url": url}
+    if https:
+        arguments["verify"] = "./ca.pem"
+
+    try:
+        response = requests.get(**arguments)
+    except ConnectionError:
+        return True
+    return response.status_code == 503
+
+
+def all_dashboards_unavailable(ops_test: OpsTest, https: bool = False) -> bool:
```

Comment:
Likewise, can we have it as `all_dashboards_available`

Reply:
We already do.... See `access_all_dashboars()`.

Reply:
@juditnovak what is the use case you need to know all dashboards are *unavailable*?